src/components/internship.ts

```diff
@@ -0,0 +1,58 @@
+import { defineComponent, computed } from 'vue'
+import Business from './Business.vue'
+
+export default defineComponent({
+  name: 'Internship',
+  components: {
+    Business,
+  },
+  props: {
+    city: {
+      required: true,
+      type: String,
+      default: 'Paris',
+    },
+    dateFrom: {
+      required: true,
+      type: Date,
+      default: '',
+    },
+    dateTo: {
+      required: true,
+      type: Date,
+      default: '',
+    },
+    business: {
+      required: true,
+      type: Object,
+      default: () => {
+        return {
+          name: '',
+          team: [{
+            name: '',
+            job: '',
+            image: '',
+            tag: '',
+          }],
+          number: 0,
+          stack: [],
+          logo: '',
+        }
+      },
+    },
```

Comment:
Quand c'est comme ca et Required, tu peux declarer un type
```suggestion
type BusinessData = {
    name: string
    team: TeamData[]
    number: number
    stack: string[]
    logo: string
}

[...]


    business: {
      required: true,
      type: Object as Proptype<BusinessData>,
    },
```